IDE/Android/user_settings.h

```diff
@@ -8,22 +8,42 @@
     #define HAVE_FIPS
 #endif
 
+#ifdef __aarch64__
+    #if !defined(__clang__) || \
+        (defined(__clang__) && defined(__clang_major__) && __clang_major__ >= 5)
+        /* older clang v4 has issue with inline assembly contraints */
+        #define WOLFSSL_ARMASM
+    #endif
+#endif
+
+#if 1 /* SP Assembly Speedups (wPAA) */
+    #define WOLFSSL_SP
+    #define WOLFSSL_SP_SMALL      /* use smaller version of code */
+    #define WOLFSSL_HAVE_SP_RSA
+    #define WOLFSSL_HAVE_SP_DH
+    #define WOLFSSL_HAVE_SP_ECC
+    #ifdef WOLFSSL_ARMASM
+        #define WOLFSSL_SP_ARM64_ASM
+    #endif
+#endif
+
 /* WPA Supplicant Support */
 #define WOLFSSL_WPAS_SMALL
 #define OPENSSL_ALL
 #define HAVE_THREAD_LS
 
 #define USE_FAST_MATH
+#define FP_MAX_BITS (4096*2) /* Maximum math bits (Max RSA key bits * 2) */
 #define TFM_TIMING_RESISTANT
 #define ECC_TIMING_RESISTANT
 #define WC_RSA_BLINDING
 
 #define HAVE_HASHDRBG
 
-#if 1
+#if 0
     #define WOLFSSL_TLS13
```

Comment:
Why don't we want TLS 1.3?

Reply:
It has to do with not wanting the WPA supplicant to use TLSv1.3. It isn't supported with this supplicant version on Android. You make a good point that it would be better to solve by changing the supplicant to not use SSLv23 or calling something else to disable TLS v1.3. The supplicant is the only thing using TLS in this customers application.

For this Android example I have gone ahead and enabled it.